src/main/java/uk/gov/justice/laa/crime/cfecrime/enums/Outcome.java

```diff
@@ -1,20 +1,20 @@
-package uk.gov.justice.laa.crime.cfecrime.cma.enums;
+package uk.gov.justice.laa.crime.cfecrime.enums;
 
-import lombok.AllArgsConstructor;
 import lombok.Getter;
 
 /**
  * Means Test Outcome values for result of assessment
  */
 @Getter
-public enum MeansTestOutcome {
+public enum Outcome {
     ELIGIBLE_WITH_CONTRIBUTION("Eligible with income contribution"),
     ELIGIBLE_WITH_NO_CONTRIBUTION("Eligible with no income contribution"),
-    INELIGIBLE("Ineligible");
+    INELIGIBLE("Ineligible"),
+    ELIGIBLE("Eligible");
```

Comment:
There should only be 3 outcomes - maybe only 2 at the moment as we don't really understand contributions

Reply:
Agreed we don't need to add a 4th oucome 'ELIGIBLE'. And I suggest we stick with the 3 outcomes (rather than 2), since that's what we have until now.

If someone is "passported" by being <18 or on the right benefit, then they pay no contribution, I believe. We figured that in this case the means test is not done, so we don't/can't calculate contribution amount. This is supported by the example in the Criminal Legal Aid Manual which says:

> Applicant is on a passporting benefit. Due to their passporting benefit status, they will not be required to make a contribution – either from income or capital – towards their defence costs.

I'll fix my flow diagrams to replace 'ELIGIBLE' with 'ELIGIBLE_WITH_NO_CONTRIBUTION', so that this is clear.